samples/benchmarks/coremark/README.rst

```diff
@@ -17,6 +17,12 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
+.. note::
+   This sample does not build or run for the ``nrf54h20dk/nrf54h20/cpuapp`` board target due to the IronSide SE migration.
+   See the ``NCSDK-34698`` in the :ref:`known_issues` page for more information.
+   This documentation page may still refer to concepts that were valid before the IronSide SE migration (for example, to he SUIT solution).
```

Comment:
```suggestion
   This documentation page may still refer to concepts that were valid before the IronSide SE migration (for example, to the SUIT solution).
```

Reply:
oops this was supposed to be removed as we don't refer to SUIT in this page